engine/crates/composition/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
```

Comment:
This is the first thing I see in this PR and I'm already into the whole thing. Very nice.